classes/person.rb

```diff
@@ -5,7 +5,7 @@ class Person < Nameable
   attr_reader :id, :type
   attr_accessor :name, :age, :rental
 
```

Comment:
- [ ] It would be better if we can run the app successfully without any errors, however, we are encountering `load_error` as seen in the screenshot below. Kindly investigate and fix
![Screenshot from 2023-03-02 13-15-17](https://user-images.githubusercontent.com/46388113/226147593-2c1d9c25-aa84-495c-ac40-06a55fcd3c47.png)